staging/src/k8s.io/apiserver/pkg/audit/context_test.go

```diff
@@ -93,25 +82,9 @@ func TestAddAuditAnnotation(t *testing.T) {
 	}
 }
 
-func TestLogAnnotation(t *testing.T) {
```

Comment:
instead of removing the test, can we rewrite it using `AddAuditAnnotation`?
```
func TestLogAnnotation(t *testing.T) {
        ctx := WithAuditContext(context.TODO())
        ev := AuditEventFrom(ctx)
	ev.Level := auditinternal.LevelMetadata
        ev.AuditID = "fake id"

	AddAuditAnnotation(ctx, "foo", "bar")
	AddAuditAnnotation(ctx, "foo", "baz")
	assert.Equal(t, "bar", ev.Annotations["foo"], "audit annotation should not be overwritten.")

	AddAuditAnnotation(ctx, "qux", "")
	AddAuditAnnotation(ctx, "qux", "baz")
	assert.Equal(t, "", ev.Annotations["qux"], "audit annotation should not be overwritten.")
}
```

Reply:
There's already a `TestAddAuditAnnotation`, so I just added a case to cover overlapping annotations.